docs/general.rst

```diff
@@ -12,17 +12,17 @@ following principles, with decreasing priority:
  3. a proper mail server offers proper security
  4. a proper mail server has a simple and beautiful user interface
 
-Mailu was designed and is kept up to date based on these. Among the most
+Mailu was designed and is kept up to date based on these. Among the 
 structuring choices, the following were quite challenging:
 
  - Mailu is based on containers (plural) so that any part can be reused
    in separate projects and that updating or swapping any component does
-   not require to alter others.
- - Mailu offers mail and does not bring any bloat in the default setup.
+   not require alterations others.
```

Comment:
This was a wrong sentence. But the new sentence is also wrong or does not read well.  Maybe change it to:
... does not require `altering other containers`.

Reply:
Fixed in b4df9407d073d1bc2f82af9bef67a28b7b6dcc79